CorePayments/src/main/java/com/paypal/android/corepayments/apis/eligibility/EligibilityAPI.kt

```diff
@@ -0,0 +1,65 @@
+package com.paypal.android.corepayments.apis.eligibility
+
+import android.content.Context
+import com.paypal.android.corepayments.CoreConfig
+import com.paypal.android.corepayments.PayPalSDKError
+import com.paypal.android.corepayments.R
+import com.paypal.android.corepayments.ResourceLoader
+import com.paypal.android.corepayments.features.eligibility.EligibilityRequest
+import com.paypal.android.corepayments.graphql.GraphQLClient
+import com.paypal.android.corepayments.optBooleanAtKeyPath
+import org.json.JSONArray
+import org.json.JSONObject
+
+internal class EligibilityAPI internal constructor(
+    private val config: CoreConfig,
+    private val graphQLClient: GraphQLClient,
+    private val resourceLoader: ResourceLoader
+) {
+    companion object {
+        const val VARIABLE_CLIENT_ID = "clientId"
+        const val VARIABLE_INTENT = "intent"
+        const val VARIABLE_CURRENCY = "currency"
+        const val VARIABLE_ENABLE_FUNDING = "enableFunding"
+    }
+
+    constructor(config: CoreConfig) :
```

Comment:
Could we combine this constructor with the main constructor by adding default values?

Reply:
Yeah def. I'll update.

Reply:
Updated here: [be3a95b](https://github.com/paypal/paypal-android/pull/268/commits/be3a95beaa29eead1dcd84488b48966d4d94e9b6)